### ISSUE TITLE
Dragging apps from Overview App Screen not working with Panel at the TopS

### ISSUE THREAD

**Issue by valiksheva** [home-sweet-gnome/dash-to-panel#2183]
**Describe the bug**
When placing Panel at the top, dragging apps to the Workspace from the app menu is not working.

Steps to reproduce:
1. Install Dash to Panel
2. Set Panel place position to Top
3. Try dragging apps from Overview App Screen to your Workspace

**Linux distribution and version**
Fedora 41

**GNOME Shell version**
46.5

**Dash-to-Panel version**
63

**Where was Dash-to-Panel installed from?**
The GNOME extensions manager

Link to screencast
https://github.com/user-attachments/assets/f4ddd5df-bc35-4877-ac61-0180904c1340

**Comment by charlesg99** [comment]
I can't reproduce this

**Comment by valiksheva** [comment]
Ok, so I tested it again on a clean Fedora setup and worked well out of the box. But the issue started to occur again after I enabled Intellihide and persisted even if disabled again.
So maybe there's a problem there.

**Comment by charlesg99** [comment]
Yes thank you, I can reproduce with intellihide enabled. I'll look into it when I get a chance

**Comment by kankaristo** [comment]
I would also appreciate it if this was possible. If the panel is on the top, and Intellihide is *off*, the panel goes on top of the search text box in the overview, so it's difficult to see what you've written. The same is true for "Show Apps".

An even better fix would be to make Gnome's overview/applications have top padding/margin, if the panel isn't hidden, but I'm not sure if an extension can even modify the overview like that.

**Comment by charlesg99** [comment]
Thanks again for reporting this, the app drag and the search text box issues should now be fixed on master.
